app/models/request.rb

```diff
@@ -1,7 +1,8 @@
 # frozen_string_literal: true
 
 class Request < ApplicationRecord
-  validates_presence_of :title, :description, :reward
+  validates_presence_of :title, :description, :reward, :status
   belongs_to :requester, class_name: 'User'
   has_many :offers
+  enum status: %i[pending active completed]
```

Comment:
Could you make the enum a hash, and save space in the numbers for "started"?

Reply:
Isn't pending request that a request waiting for a helper ?

Reply:
I meant "started" as in requester presses start in order to lock the job and make the request undeletable. But maybe we can just shove it in when we get to it.

Reply:
done